packages/validator/src/validator.ts

```diff
@@ -134,6 +134,19 @@ export class Validator {
       this.clock.start(this.controller.signal);
       this.chainHeaderTracker.start(this.controller.signal);
 
+      // Add notifier to warn user if primary node is unhealthy as there might
```

Comment:
Something to consider is a dedicated notifier as we have it in the beacon node, this would allow to emit further information every epoch / slot, for example the statuses of imported validators / active keys, etc.

Reply:
That would be nice